setup.py

```diff
@@ -11,23 +11,24 @@
 # FOR A PARTICULAR PURPOSE.
 #
 ##############################################################################
-"""Setup
-"""
+version = '1.6.0'
```

Comment:
Looks like 4e4726219c6be20513122ce53c1164ac17354fb7 should be pushed directly to `master`, with the `1.6.0` tag assigned to it.